stdlib/io/src/main/java/org/ballerinalang/stdlib/io/utils/PropertyUtils.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.stdlib.io.utils;
+
+import org.ballerinalang.jvm.StringUtils;
+import org.ballerinalang.jvm.types.BMapType;
+import org.ballerinalang.jvm.types.BTypes;
+import org.ballerinalang.jvm.values.MapValue;
+import org.ballerinalang.jvm.values.MapValueImpl;
+import org.ballerinalang.jvm.values.MappingInitialValueEntry;
+import org.ballerinalang.jvm.values.api.BMap;
+import org.ballerinalang.jvm.values.api.BString;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.util.Enumeration;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+
+/**
+ * The utility class used to execute read/write operations to a property file.
+ *
+ */
+public class PropertyUtils {
+    private static final Logger log = LoggerFactory.getLogger(PropertyUtils.class);
+    private static final BMapType mapType = new BMapType(BTypes.typeString);
+    private static Properties properties = null;
```

Comment:
Let's say I have to read 2 properties file in single ballerina code. Due to this only the first reader will be successful isn't it? We have to load properties every time from the reader.

Reply:
No, when I test it is the other way around. Actually I added this to solve the problem that you mentioned. For example, once we read we cannot use the `reader` once, it is not reusable. This won't be a problem in other APIs since they read only once. From, this we store the properties statically and reuse again when we needed.

Reply:
Here we are not reading the properties from the same file. If we try to read 2 different files, this will read only from one of this since the Properties is a static variable.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/24978/commits/fe3ba68dad14e44cee577fa380117148c65dc4d9